proto/src/determined/experiment/v1/experiment.proto

```diff
@@ -106,6 +106,12 @@ message PatchExperiment {
   google.protobuf.StringValue name = 4;
   // The experiment notes.
   google.protobuf.StringValue notes = 5;
+  // The maximum resources in slots.
+  google.protobuf.Int32Value max_slots = 6;
```

Comment:
this using nullable fields for patches thing is mostly nice, but what if we need to set a field to nil? why did we move to this from field masks (i may've just forgotten)?